apps/aecore/src/aec_conductor.erl

```diff
@@ -507,18 +507,19 @@ preempt_if_new_top(#state{ top_block_hash = OldHash,
     case aec_chain:top_block_hash() of
         OldHash -> no_change;
         NewHash ->
-            ok = aec_tx_pool:top_change(OldHash, NewHash),
+            {ok, NewBlock} = aec_chain:get_block(NewHash),
```

Comment:
We had `Block` where we called `preempt_if_new_top` maybe pass it instead?

Reply:
I tried. It wasn't a 'real' block, so I decided to play it safe.

Reply:
If only Thomas had refactored the code the way some reviewer suggested it would have been a real block ;-)